### ISSUE TITLE
[Question] memory saving and batch_size

### ISSUE THREAD

**Issue by nevermet** [Lightning-AI/litgpt#704]
Dear all,

When I posted a question about lora finetuning, I got the answer that it'll be helpful to set micro_batch_size to 1 to save GPU's memory and it will not affect the finetuning performance. 

Then, how about the variable 'batch_size'?
Some people say they lowered the number to run the finetuning with limited GPU's memory. Is it ok to lower the number? Does it affect the finetuning performance?

Thanks in advance.

**Comment by Andrei-Aksionov** [comment]
Hello @nevermet 

With `micro_batch_size` you "virtually increase" the batch size. With default setting it's an increase from 4 to 128. That's why it should not affect fine-tuning performance.

If you change the `batch_size` it can affect the performance, as it affects the gradients update (more noise). Which is not necessary a bad thing, depending on the data/model/batch_size of course. 
So you can definitely try this out. Only do not forget to change the learning rate [accordingly](https://stackoverflow.com/questions/53033556/how-should-the-learning-rate-change-as-the-batch-size-change).

**Comment by rasbt** [comment]
I hope this question was successfully addressed. But please feel free to reopen with any follow-up questions or concerns!
